font-codegen/src/parsing.rs

```diff
@@ -235,12 +235,37 @@ pub(crate) struct FieldReadArgs {
     pub(crate) inputs: Vec<syn::Ident>,
 }
 
+#[derive(Clone, Debug)]
+pub(crate) enum CountArg {
```

Comment:
Nit: suggest examples in comment

Reply:
Also ".." for all is perhaps a touch Rust'ish?

Count(*)  // I can guess what this means
Count(..)  // Python hat on, whats this?

Reply:
(belatedly realizes the examples are right below) ... maybe just contemplate the .. then :)

Reply:
the attribute syntax that we use here doesn't need to be universal, it just needs to map to the schema; this is all going to get turned into yaml, where we will just use 'all' or something.